cmd/contour/serve.go

```diff
@@ -418,8 +428,8 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 
 		switch ctx.XDSServerType {
 		case "contour":
-			rpcServer := xds.RegisterServer(
-				xds.NewContourServer(log, contour.ResourcesOf(resources)...),
+			grpcServer := contourxds.RegisterServer(
+				contourxds.NewContourServer(log, contour.ResourcesOf(resources)...),
```

Comment:
Suggest that we hoist a `*grpc.Server`  variable out of the switch and then use the same listen code for both kinds of server, rather than duplicating it.

Reply:
👍